package.json

```diff
@@ -35,16 +35,17 @@
   "license": "MIT",
   "repository": "jest-community/jest-extended",
   "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-transform-async-to-generator": "^7.0.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0",
```

Comment:
we don't need any of these when we use the `env` preset

Reply:
Are they still needed to be mentioned in the plugins list? Or does that come through present env also? I wasn't totally certain.

```json
"plugins": [
  "@babel/plugin-transform-modules-commonjs",
  "@babel/plugin-proposal-object-rest-spread",
  "@babel/plugin-transform-async-to-generator",
  "module:babel-jest-assertions",
  "gwt",
  "macros"
]
```

Reply:
Remove all mention of them, they're included in the env preset 🙂

Reply:
Awesome. Thanks for confirming. Removed.